pyproject.toml

```diff
@@ -45,12 +45,12 @@ isort = "^5.10.1"
 pytest-cov = "^4.0.0"
 pytest-asyncio = "^0.19.0"
 nox = "^2022.1.7"
-nox-poetry = "^1.0.1"
+nox-poetry = "^1.0.2"
```

Comment:
`bump2version` changed these because `nox-poetry` is on `1.0.1` and so is `bump2version`. Hilarious whoopsies. https://pypi.org/project/bump2version/#history